suzieq/engines/pandas/arpnd.py

```diff
@@ -45,15 +51,40 @@ def summarize(self, **kwargs):
         """Summarize ARPND info across namespace"""
         self._summarize_on_add_field = [
             ('deviceCnt', 'hostname', 'nunique'),
-            ('arpNdEntriesCnt', 'ipAddress', 'count'),
             ('macaddrCnt', 'macaddr', 'count'),
             ('oifCnt', 'oif', 'count'),
-            ('uniqueOifCnt', 'oif', 'nunique')]
+            ('uniqueOifCnt', 'oif', 'nunique'),
+            ('arpNdEntriesCnt', 'ipAddress', 'count')]
 
         self._summarize_on_add_with_query = [
-            ('remoteEntriesCnt', 'state == "remote"', 'ipAddress'),
-            ('staticEntriesCnt', 'state == "permanent"', 'ipAddress'),
-            ('failedEntryCnt', 'state == "failed"', 'ipAddress')]
+            ('arpNdV4EntriesCnt', '@self._check_ipvers(ipAddress, 4)',
+                'ipAddress'),
```

Comment:
Calling it arpNdv4 makes less sense than arpEntriesCnt and v6NDEntriesCnt or v6NeighDiscEntriesCnt. The same for the other entries with similar prefix. Could you please fix it?